clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ServerStoreProxy.java

```diff
@@ -58,4 +68,56 @@ default Chain compact(Chain chain, long hash) {
    */
   void close();
 
+  interface Entry extends Chain {
```

Comment:
This interface name is confusing. I would prefer a name that reminds that this still is a `Chain`, like `MorePowerfulChain`, `UltraChain`, `ExtendedChain`...

Reply:
Fixed... chose `ChainEntry`... chain because it's a chain, entry because it's 'bound' to the store and represents an entry in that store.